browser/src/map/Map.js

```diff
@@ -292,6 +292,8 @@ window.L.Map = window.L.Evented.extend({
 				if (window.ThisIsTheAndroidApp) {
 					window.postMobileMessage('hideProgressbar');
 				}
+
+				app.activeDocument.activeView.sendClientVisibleArea(true);
```

Comment:
there is no possibility to lack some of these? *activeView activeDocument at this point) ?
asking just to be sure

Reply:
I mean: are we sure they are not undefined at this place in the code? :)